src/main/java/net/pms/util/UMSUtils.java

```diff
@@ -171,13 +171,22 @@ private static int getHW(String[] s, int pos) {
 		}
 	}
 
+	/**
+	 * JPEG_TN   size must not exceed  160x160 (horizontal * vertical)
+	 * JPEG_SM   size must not exceed  640x480
+	 * JPEG_MED size must not exceed 1024x768
+	 * JPEG_LRG size must not exceed 4096x4096
+	 * @param in
+	 * @param r
+	 * @return
+	 * @throws IOException
+	 */
 	@SuppressWarnings("deprecation")
```

Comment:
I think this function can be deprecated after this change so please remove line 184 and add @deprecated to the docblock